src/test/java/ParserTest.java

```diff
@@ -0,0 +1,15 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
```

Comment:
Not sure if your `checkstyle.xml` validates your checkstyle for your Test classes - but just leaving a note here, in case you might need JavaDoc here!